perfmetrics/scripts/testing_on_gke/examples/dlio/parse_logs.py

```diff
@@ -304,29 +288,38 @@ def writeOutput(
           )
           continue
 
-        new_row = (
-            record_set["mean_file_size"],
-            record_set["num_files_train"],
-            total_size,
-            record_set["batch_size"],
-            scenario,
-            r["epoch"],
-            r["duration"],
-            r["train_au_percentage"],
-            r["train_throughput_samples_per_second"],
-            r["train_throughput_mb_per_second"],
-            r["throughput_over_local_ssd"],
-            r["lowest_memory"],
-            r["highest_memory"],
-            r["lowest_cpu"],
-            r["highest_cpu"],
-            r["pod_name"],
-            r["start"],
-            r["end"],
-            f'"{r["gcsfuse_mount_options"].strip()}"',  # need to wrap in quotes to encapsulate commas in the value.
-            args.instance_id,
-        )
-        rows.append(new_row)
+        try:
```

Comment:
As discussed offline. This was needed because some files failed parsing, and this was breaking the flow here. Anyway, I moved this changed to the parent PR i.e. #2524 where this change was more relevant.

Reply:
please add code comment why and how we will remove the try catch in future. THanks.